.github/workflows/py3.yml

```
@@ -22,8 +22,11 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
-      - name: Install pynetbox and testing packages.
-        run: pip install . black==19.10b0 pytest pytest-docker faker
+      - name: Install pynetbox
+        run: pip install .
+
+      - name: Install dev requirements
+        run: pip install -r requirements-dev.txt
```

I'm cool with just installing pynetbox under dev requirements

```suggestion
      - name: Install dev requirements
        run: pip install -r requirements-dev.txt .
```

Should do it.